train.py

```diff
@@ -153,6 +152,15 @@ def main(binary, mode, config, model_path, extra, partition, comment, ngpu, gpu1
             print('Skipping')
             return
 
+        # Backup the config file to sub_log_dir
+        config_copy = os.path.join(
+            sub_log_dir, os.path.basename(config)) + '-' + timestamp
+        shutil.copy(config, config_copy)
```

Comment:
Can we update the old config file with the new flags in `--extra`? This will decouple the script from the `--extra` in anymore and make the program always use the config file. You need to change this for `continue` mode as well by updated the saved flags.

Reply:
> Can we update the old config file with the new flags in --extra

Did you mean update the *new* config file ?

Reply:
yes, when you save/write the flag files, you can update it with the new flags in `--extra`.